config/config.go

```diff
@@ -36,6 +36,9 @@ type Config struct {
 	EncryptionDisabled           bool          `envconfig:"ENCRYPTION_DISABLED"`
 	StopConsumingOnUnhealthy     bool          `envconfig:"STOP_CONSUMING_ON_UNHEALTHY"`
 	S3PublicURL                  string        `envconfig:"S3_PUBLIC_URL"`
+	OTExporterOTLPEndpoint       string        `envconfig:"OTEL_EXPORTER_OTLP_ENDPOINT"`
+	OTServiceName                string        `envconfig:"OTEL_SERVICE_NAME"`
+	OTBatchTimeout               time.Duration `envconfig:"OTEL_BATCH_TIMEOUT"`
```

Comment:
we're probably not all that fussy about controlling the timeout value - it defaults to 5 sec anyway